packages/logs/lib/models/logContextGetter.ts

```diff
@@ -54,7 +56,10 @@ export const logContextGetter = {
     async get({ id }: { id: MessageRow['id'] }, options?: Options): Promise<LogContext> {
         try {
             if (envs.NANGO_LOGS_ENABLED) {
-                const operation = await getOperation({ id });
+                const store = await getKVStore();
+                const indexName = await store.get(`es:operation:${id}:indexName`);
+
+                const operation = await getOperation({ id, indexName });
                 return new LogContext({ parentId: id, operation }, options);
```

Comment:
do you know how much load is it going to add on redis?

Reply:
not really, the keys are short lived and with the current rate of actions it's going to be less than 200 keys per minutes

Reply:
No impact on Redis (surprising actually)
<img width="1074" alt="Screenshot 2024-06-03 at 10 55 00" src="https://github.com/NangoHQ/nango/assets/1637651/7c629816-efef-4c82-bdc7-6e0759696ef9">


But almost halved CPU for ES 🙌🏻 
<img width="1282" alt="Screenshot 2024-06-03 at 10 54 41" src="https://github.com/NangoHQ/nango/assets/1637651/04d33371-58c8-46bd-8cf4-fc8b84b181b4">

Reply:
nice!